caikit/runtime/servicers/global_train_servicer.py

```diff
@@ -98,13 +103,15 @@ def __init__(self, training_service: ServicePackage):
         )
         super()
 
-    def Train(self, request, *_, **__) -> TrainingJob:
+    def Train(self, request, context, *_, wait=False, **__) -> TrainingJob:
```

Comment:
When would we ever pass `wait` to this function? This is the rpc handler, so afaik we never actually call it directly?

Reply:
This one is getting called for test to keep the connection open: https://github.com/gkumbhat/caikit/blob/add_train_cancel_handler/tests/runtime/servicers/test_global_train_servicer_impl.py#L438

Reply:
I think this needs to get removed per slack conversation